src/core/router/identity.cc

```diff
@@ -730,27 +730,23 @@ Keys CreateRandomKeys() {
 IdentHash CreateRoutingKey(
     const IdentHash& ident) {
   std::uint8_t buf[41];  // ident + yyyymmdd
-  memcpy(buf, (const std::uint8_t *)ident, 32);
+  std::memcpy(buf, (const std::uint8_t *)ident, 32);
   time_t t = time(nullptr);
   struct tm tm;
-  // TODO(unassigned): never use sprintf, use snprintf instead.
+  std::stringstream ss;
+  ss.imbue(std::locale(std::locale::classic()));
```

Comment:
>ss.imbue(std::locale(std::locale::classic()));

Why do you want this? I understand conceptually why but this doesn't appear necessary (and the router works fine without it in this branch).

Reply:
> this doesn't appear necessary

I realize now that this is the default, but wanted to ensure UTC  with no localization leaks before.